analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Invocations.cs

```diff
@@ -183,34 +240,124 @@ public void Instance_InstanceMethodCallDoesNotClearFieldsOnOtherInstances(string
             var code = $@"
 public class Sample
 {{
-    object field1 = null;
-    object field2 = null;
-    static object staticField1 = null;
-    static object staticField2 = null;
+    object field1;
+    static object staticField1;
 
-    void CallToMethodsShouldResetFieldConstraints()
+    void Main()
     {{
-        field1 = new object();
-        field2 = new object();
-        staticField1 = new object();
-        staticField2 = new object();
+        field1 = null;
+        staticField1 = null;
```

Comment:
This scaffolding can also be simplified (like the one above). Just using `InstanceMethod` instead of `DoSomething.